### ISSUE TITLE
Should 1.1.1 have been 1.2.0?

### ISSUE THREAD

**Issue by evgeni** [dm-drogeriemarkt/foreman_netbox#100]
Looking at the diff https://github.com/dm-drogeriemarkt/foreman_netbox/compare/1.1.0...1.1.1 I it changes the minimum Foreman version, so that doesn't sound like a "patch" release? :)

**Comment by kamils-iRonin** [comment]
indeed, @danielwieberneit can we release `1.2.0` instead of `1.1.1`? there is also a mention of `1.2` in the compatibility matrix  https://github.com/dm-drogeriemarkt/foreman_netbox/blob/master/README.md?plain=1#L13
